tests/sentry/auth/test_access.py

```diff
@@ -22,6 +22,7 @@ def test_no_access(self):
         assert not result.has_team_access(team)
         assert not result.has_team_scope(team, 'project:read')
         assert not result.has_project_access(project)
+        assert not result.has_projects_access([project])
```

Comment:
Should there be a test for this method that covers checking multiple projects?

Reply:
Good point, added